frontend/src/components/MainDashboard/DashboardBody/DashboardBody.js

```diff
@@ -138,28 +188,61 @@ export default class DashboardBody extends Component {
             <div className="col-md-9">
               <div className="card my-4">
                 <div className="container">
-									<div className="row">
-										{data ? (
-											data.map(pet => {
-												return (
-													<PetCard
-														name={pet.name}
-														species={pet.species}
-														breed={pet.breed}
-														dob={pet.dob}
-														gender={pet.gender}
-														description={pet.description}
-													/>
-												)})
-											) : ( null )
-										}
-									</div>
+                  <div className="row">
+                    {data
+                      ? data
+                          .filter(pet => {
+                            //filter based on species
+                            if (
```

Comment:
Nitpick: 

looks like it's checking to make sure dog !== cat. This can be a bit cleaner with 

```
if (this.state.dog && !this.state.cat) { return ...dog }
else if (this.state.cat && !this.state.dog) { return ...cat }
else { return pet; }
```
A bit less redundancy this way.